src/Chapter3.hs

```diff
@@ -910,6 +1011,21 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold {
+    unGold :: Int
+  }
+instance Append Gold where
+    append :: Gold -> Gold -> Gold
+    append g h = Gold ((unGold g) + (unGold h))
+
+instance Append [a] where
+    append :: [a] -> [a] -> [a]
+    append g h = g ++ h
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```